src/vaadin-avatar-group.html

```diff
@@ -166,7 +168,11 @@
                   activeUsers: {
                     one: 'Currently one active user',
                     many: 'Currently {count} active users'
-                  }
+                  },
+                  // Screen reader announcement when user joins group.
+                  joined: '{user} joined',
+                  // Screen reader announcement when user leaves group.
+                  left: '{user} left'
```

Comment:
Nit: we could emphasize that `{user}`is the keyword that will be used and explain that it will be replaced with `name / abbr / anonymous`.

Reply:
Done.